### ISSUE TITLE
Tapping on titles in the "explore" tab takes me back to the main menu

### ISSUE THREAD

**Issue by qwuzzy** [zend10/AL-chan#113]
Using the "explore" feature and trying to click on an anime or manga doesn't bring up the page for that title and instead exits the menu.
 https://github.com/zend10/AL-chan/assets/32120669/928d89ae-fe75-40e0-9ab2-f445d9baac11

**Comment by MegaBluejay** [comment]
No guarantees it doesn't break anything, but this seems to fix the issue

```diff
diff --git a/app/src/main/java/com/zen/alchan/ui/home/HomeFragment.kt b/app/src/main/java/com/zen/alchan/$
index a6c656f8..02df8c44 100644
--- a/app/src/main/java/com/zen/alchan/ui/home/HomeFragment.kt
+++ b/app/src/main/java/com/zen/alchan/ui/home/HomeFragment.kt
@@ -59,7 +59,9 @@ class HomeFragment : BaseFragment<FragmentHomeBinding, HomeViewModel>() {
             },
             viewModel.searchCategoryList.subscribe {
                 dialog.showListDialog(it) { data, _ ->
-                    navigation.navigateToExplore(data)
+                    navigation.navigateToExplore(data, null) {
+                        it()
+                    }
                 }
             }
         )
```

I also removed this line to make the back button go back to explore from a media, rather than the previous screen

```diff
diff --git a/app/src/main/java/com/zen/alchan/ui/explore/ExploreFragment.kt b/app/src/main/java/com/zen/a$
index 07f66180..2f9301e7 100644
--- a/app/src/main/java/com/zen/alchan/ui/explore/ExploreFragment.kt
+++ b/app/src/main/java/com/zen/alchan/ui/explore/ExploreFragment.kt
@@ -183,7 +183,6 @@ class ExploreFragment : BaseFragment<FragmentExploreBinding, ExploreViewModel>()

     private fun navigateToBrowseScreen(navigation: () -> Unit) {
         listener?.let {
-            goBack()
             it.doNavigation { navigation() }
         } ?: kotlin.run {
             navigation()
```

**Comment by zend10** [comment]
Thanks for the report. Noticed it very late. Fixed in 1.9.7.
